UnitTests/Tests/TRWeatherUpdateSpec.m

```diff
@@ -1,6 +1,6 @@
 @import CoreLocation;
 #import "TRWeatherUpdate.h"
-#import "TRTemperature.h"
+#import "UnitTests-Swift.h"
```

Comment:
Yes it is. This is umbrella header which contains headers for `Temperature` class for Objective-C.